app/src/main/java/com/example/fizzbuzz/fizzbuzz.kt

```diff
@@ -1,5 +1,70 @@
 package com.example.fizzbuzz
 
+fun addWord(words: MutableList<String>, word: String) {
+    words.add(word)
+}
+
+fun setWordsToBong(words: MutableList<String>) {
+    words.clear()
+    words.add("Bong")
+}
+
+fun insertFezz(words: MutableList<String>) {
+    var inserted = false
+    for (i in words.indices) {
+        if (words[i][0] == 'B') {
+            words.add(i, "Fezz")
+            inserted = true
+            break
+        }
+    }
+    if (!inserted) {
+        words.add("Fezz")
+    }
+}
+
+fun reverseWords(words: MutableList<String>) {
+    words.reverse()
+}
+
+fun printNumOrWords(num: Int, words: MutableList<String>) {
```

Comment:
This method is a little more convoluted than it needs to be. You could simplify it by using `jointostring` and handling both cases at once.
i.e. something like
```
fun printNumOrWords(num: Int, words: MutableList<String>) {
    println(if (words.isEmpty()) num else words.joinToString(""))
}
```